api/leaderworkerset/scheme/scheme.go

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
please update the year and add a comment that this is adapted from controller-runtime and that we are doing this to cut the api pkg dependency on controller-runtime